.github/workflows/test.yaml

```diff
@@ -0,0 +1,68 @@
+name: "🧪 Test"
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+  # build weekly at 4:00 AM UTC
+  schedule:
+    - cron: '0 4 * * 1'
+
+jobs:
+  test:
+    name: "${{ matrix.name }}"
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: "Linux"
+            runner: "ubuntu-latest"
+            cpythons:
+              - "3.8"
+              - "3.9"
+              - "3.10"
+              - "3.11"
+              - "3.12"
+            tox-post-environments:
+              - "py3.8-mindeps"
+              - "py3.8-sdkmain"
+              - "py3.12-sdkmain"
```

Comment:
Should this one not also set `tox-environments-from-pythons: true`?
I'm somewhat surprised to see it set for the other platforms but not for this one.

Reply:
`tox-environment-from-pythons` is automatically enabled if any of the following keys are configured:

* `tox-pre-environments`
* `tox-post-environments`
* `tox-factors`

Since none of those are configured for the other jobs in the matrix, those other configurations need that to be specified.